### PR TITLE
3.0 preview dev

### PR DESCRIPTION
**Pull request by notwaldorf**
(this is a bit of a weirdo element, so I'd like some more pair of eyes to make sure I didn't mess up)

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -18,64 +18,53 @@
...
-$_documentContainer.innerHTML = `<div class="vertical-section-container centered">
+  <div class="vertical-section-container centered">
     <p><b>Note</b>: in order to hear the announcements, be sure to turn on your favorite screen reader!</p>
 
     <h4>Announcer used directly on the main document</h4>
     <demo-snippet>
       <template>
         <input id="input" value="Pizza is delicious">
-        <paper-button raised="" onclick="_announce()">Announce</paper-button>
+        <paper-button raised="" onclick="window._announce()">Announce</paper-button>
```
Comment (e111077) [inline demo/index.html]: can you please check if this works without `window.`? If it does not, then can you put this in the README?

**Review by e111077 — CHANGES_REQUESTED** [review]

File: demo/index.html (hunk 1 above)
Comment (notwaldorf) [reply demo/index.html]: Done! And added a known issues section

**Review by e111077 — APPROVED** [review]
gonna want to format. preapproval

File: test/iron-a11y-announcer.html [hunk 2]
```diff
@@ -13,42 +13,38 @@
 <head>
   <title>iron-a11y-announcer</title>
 
-  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
-  <script src="../../../wct-browser-legacy/browser.js"></script>
-
-  <script type="module" src="../../polymer/polymer-legacy.js"></script>
-  <script type="module" src="../iron-a11y-announcer.js"></script>
+  <script src=" ../node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+  <script src=" ../node_modules/wct-browser-legacy/browser.js"></script>
```
Comment (bicknellr) [inline test/iron-a11y-announcer.html]: There's an extra space in front of these URLs. I'm not sure if it matters though.

File: test/index.html [hunk 3]
```diff
@@ -9,13 +9,13 @@
 --><html><head>
     <meta charset="utf-8">
     <title>iron-a11y-announcer tests</title>
-    <script src="../../../wct-browser-legacy/browser.js"></script>
+    <script src=" ../node_modules/wct-browser-legacy/browser.js"></script>
```
Comment (bicknellr) [inline test/index.html]: Also an extra space.

**Review by bicknellr — APPROVED** [review]
LGTM with a couple pedantic comments.

**Pull request merged**